velox/exec/GroupingSet.cpp

```diff
@@ -742,7 +742,7 @@ bool GroupingSet::getOutput(
       : 0;
   if (numGroups == 0) {
     if (table_ != nullptr) {
-      table_->clear();
+      table_->clear(/*freeTable=*/true);
```

Comment:
Can we also check if HashBuild needs this change (putting true to clear() method)?

Reply:
I think it is more intuitive to have HashTable::clear() take true as default instead of false.